src/pages/thankyou.tsx

```diff
@@ -4,19 +4,21 @@ import Layout from '../layout/Layout';
 import { FEEDBACK_LINK } from '../constants';
 import { getUuidFromCookies, deleteAllCookiesOnServerSide } from '../utils';
 
-const title = 'Thank You - Fares data build tool';
-const description = 'Thank you page for the Fares Data Build Tool';
+const title = 'Thankyou - Fares Data Build Tool';
```

Comment:
I would keep thank you as two separate words

Reply:
Reverted back to having these as separate words